tools/ldb_cmd.cc

```diff
@@ -4196,7 +4196,11 @@ void UnsafeRemoveSstFileCommand::DoCommand() {
     VersionEdit edit;
     edit.SetColumnFamily(cfd->GetID());
     edit.DeleteFile(level, sst_file_number_);
-    s = w.LogAndApply(cfd, &edit);
+    std::unique_ptr<FSDirectory> db_dir_;
+    assert(options_.env->GetFileSystem()
+               ->NewDirectory(db_path_, IOOptions(), &db_dir_, nullptr)
+               .ok());
+    s = w.LogAndApply(cfd, &edit, db_dir_.get());
```

Comment:
Fine with me except the `assert()` and `db_dir_` is named like an instance variable.

Reply:
Fixed